pkg/reconciler/configuration/configuration.go

```diff
@@ -196,6 +201,18 @@ func (c *Reconciler) reconcile(ctx context.Context, config *v1alpha1.Configurati
 		return err
 	}
 
+	if (rc == nil || rc.Status != corev1.ConditionTrue) && sortedRevisions != nil {
+		// Update the LatestReadyRevisionName to the last ready revision
+		for _, rev := range sortedRevisions {
+			if rev != nil {
+				if rr := rev.Status.GetCondition(v1alpha1.RevisionConditionReady); rr != nil && rr.Status == corev1.ConditionTrue {
```

Comment:
`rev.Status.IsReady()`?

Reply:
I was thinking if we should match it with [the previous check](https://github.com/knative/serving/blob/master/pkg/reconciler/configuration/configuration.go#L169). Which one do you prefer?